admin/handlers.php

```diff
@@ -1006,6 +1006,28 @@ function gigpress_empty_trash() {
 	
 }
 
+// HANDLER: RESTORE SHOW
+// ======================
+
+function gigpress_restore_show() {
+
+	global $wpdb;
+	$wpdb->show_errors();
+	check_admin_referer('gigpress-action');
+
+	$show = [ 'show_status' => 'active' ];
+	$where = array( 'show_id' => absint( $_GET['show_id'] ) );
+
+	$restore_show = $wpdb->update( GIGPRESS_SHOWS, $show, $where );
+
+	if( $restore_show ) { ?>
+		<div id="message" class="updated fade"><p><?php esc_html_e( 'The selected show has been restored.', 'gigpress' ); ?></p></div>
```

Comment:
Maybe move the html outside the conditional or set the string into a var via ternary and display it via variable without the if?

Reply:
Keeping as is for now to keep it simple